### ISSUE TITLE
Request tweak

### ISSUE THREAD

**Issue by 7o77** [NoW4U2Kid/NoW4U2Kid.github.io#1]
Can You add localiapstore in your repo ( dopamine jailbreak ) and thanks 🖤

**Comment by NoW4U2Kid** [comment]
No, because it's already on another repo and also just use Satella from Paisseon's repo

**Comment by 7o77** [comment]
I try to use it and when I purchase and cancel
Like I didn't do anything nothing happened):
________________________________

**Comment by NoW4U2Kid** [comment]
It works fine for me, maybe your settings aren't correct. Or maybe the app you are usin on has server-sided verification.

**Comment by 7o77** [comment]
Can you try for OVF editor app? https://apps.apple.com/us/app/ovf-editor/id1478225590
This app for upload for snapchat see if the ads will gone or no and tell me
Btw this is my settings:


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by NoW4U2Kid** [comment]
Could you contact me on discord please? It will be easier that way. Cypwn discord.

**Comment by 7o77** [comment]
I can’t see you
Send your name and #
My name No-X#7840

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
